wkb.hpp

```diff
@@ -230,6 +230,10 @@ class parser_t
             }
         }
 
+        if (out[0] != Endian)
+            throw std::runtime_error(
+                "Endian setting of database WKB not supported.");
+
```

Comment:
Or better yet, something like "Different endian settings of database and osm2pgsql are not supported. Database is big-endian, osm2pgsql is little-endian."